docs/components/navigation.css

```diff
@@ -1,28 +1,37 @@
-.navigation {
+.navigation ul {
+  list-style-type: none;
+  padding-inline-start: 0px;
 }
 
-.navigation h4 {
-  margin-left: 10px;
+.navigation ul:not(.depth0) {
+  padding-inline-start: 8px;
 }
 
-.sidebar > .bp3-menu {
-  background-color: unset;
+.navigation li {
+  padding: 0px 8px;
+  border-radius: 8px;
 }
 
-.menu {
-  color: #2f2d44;
+.navigation li.depth0.hasChildren {
```

Comment:
Many of these styles are quite specific and brittle. Can they each be a class and put directly on the element?